cmd/algorand-indexer/daemon.go

```diff
@@ -141,6 +145,7 @@ func init() {
 	daemonCmd.Flags().StringVarP(&metricsMode, "metrics-mode", "", "OFF", "configure the /metrics endpoint to [ON, OFF, VERBOSE]")
 	daemonCmd.Flags().DurationVarP(&writeTimeout, "write-timeout", "", 30*time.Second, "set the maximum duration to wait before timing out writes to a http response, breaking connection")
 	daemonCmd.Flags().DurationVarP(&readTimeout, "read-timeout", "", 5*time.Second, "set the maximum duration for reading the entire request")
+	daemonCmd.Flags().Uint32VarP(&maxConn, "max-conn", "", 0, "set the maximum connections allowed in the connection pool")
```

Comment:
Is this accurate?
```suggestion
	daemonCmd.Flags().Uint32VarP(&maxConn, "max-conn", "", 0, "set the maximum connections allowed in the connection pool, if the maximum is reached subsequent connections will be queued and wait until a connection becomes available, or timeout according to the read-timeout setting")
```

Reply:
The subsequent connections I believe will block, which is a little different than queued.  But the rest seems accurate.

Reply:
"queued and wait" is kinda the same thing as block? How about just saying wait?

Reply:
Sure